maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/HardshipOrchestrationService.java

```diff
@@ -43,16 +48,18 @@ public ApplicationDTO create(WorkflowRequest request) {
             // Need to refresh from DB as HardshipDetail ids may have changed
             HardshipReviewDTO newHardship = hardshipService.find(performHardshipResponse.getHardshipReviewId());
 
-            CourtType courtType = request.getCourtType();
-            if (courtType == CourtType.MAGISTRATE) {
-                hardshipOverview.setMagCourtHardship(newHardship);
-                if (isAssessmentComplete(newHardship.getAsessmentStatus())) {
-                    application = processMagCourtHardshipRules(request);
+            switch (courtType) {
+                case MAGISTRATE -> {
+                    hardshipOverview.setMagCourtHardship(newHardship);
```

Comment:
How come we need to create a new HardshipOverviewDTO object and add the courtType to it? I can't see anywhere in this method  that the hardshipOverview is used, or is this just in place to support future changes?

Reply:
We are updating the hardshipOverview object in the workflow request for further processing. I have refactored the code to improve the readability.